app/api/schema/faq_types.py

```diff
@@ -0,0 +1,37 @@
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema, Relationship
+
+from app.api.helpers.utilities import dasherize
+
+
+class FaqTypeSchema(Schema):
+    """
+    Api Schema for faq type model
+    """
+
+    class Meta:
+        """
+        Meta class for FaqTypeSchema
+        """
+        type_ = 'faq-type'
+        self_view = 'v1.faq_type_detail'
+        self_view_kwargs = {'id': '<id>'}
+        inflect = dasherize
+
+    id = fields.Str(dump_only=True)
+    name = fields.Str(required=True)
+    event = Relationship(attribute='event',
```

Comment:
Can't there be many events having same faq-types?

Reply:
No, faq types will be defined by the event organiser for that specific event only